index.html

```diff
@@ -83,25 +83,29 @@ <h2 class="header-subtitle text-white font-inter">
         <h2 class="works-title text-black font-crete-round">
           My Recent Works
         </h2>
-        <hr />
+        <div>
```

Comment:
- [ ] Great job, congratulation, you have done a great job, I personally appreciate your work 💪🏻 , I think we can make this better by adding some few things, on hover action for each card, please make those texts on the image display none and make the see project display flex, you designed it already but not in a good way, let me show you with an image,
<img width="656" alt="hover" src="https://user-images.githubusercontent.com/79058364/154554837-f300bb54-bea9-4de6-83dc-af5c66da423a.png">
😄  hope this makes sense, so please implement a hover action make those text and capacity display:none and display flex the see project button

Reply:
Hi, I'm planing to add these functionalities in my next task